.github/pull_request_template.md

```diff
@@ -2,17 +2,17 @@
 - https://github.com/RedHatInsights/secure-coding-checklist
 
 ## Secure Coding Checklist
```

Comment:
It would be nice to have a reference to what the items in list actually mean.

```suggestion
## [Secure Coding Checklist](https://owasp.org/www-pdf-archive/OWASP_SCP_Quick_Reference_Guide_v2.pdf)

```

Reply:
or maybe better

```suggestion
## [Secure Coding Checklist](https://owasp.org/www-project-secure-coding-practices-quick-reference-guide/migrated_content#)
```

Reply:
Maybe in a followup :wink:

Reply:
Nevermind, we're already pointing to https://github.com/RedHatInsights/secure-coding-checklist.